partcad-cli/src/partcad_cli/click/commands/init.py

```
@@ -7,30 +7,148 @@
-import os
+import os, sys
+from packaging import version
+
 import partcad.logging as logging
 from partcad.globals import create_package
+from partcad.consts import ROOT
+from partcad import __version__ as pc_version
+
 
+class DynamicPromptOption(click.Option):
+    def prompt_for_value(self, ctx):
```

Ruff (0.8.2)

Missing return type annotation for public function `prompt_for_value`

(ANN201)